inference-engine/src/vpu/common/include/vpu/ngraph/utilities.hpp

```diff
@@ -15,6 +20,60 @@ std::shared_ptr<ngraph::Node> shapeToConstant(const ngraph::element::Type& type,
 
 std::shared_ptr<ngraph::Node> gatherShapeElements(const ngraph::Output<ngraph::Node>&, int startIndex, size_t elemCount);
 
-void printTo(std::ostream& stream, const ngraph::NodeTypeInfo& object);
+template<>
+inline void printTo(std::ostream& stream, const ngraph::NodeTypeInfo& object) {
+    stream << object.name << " ver. " << object.version;
+}
```

Comment:
I still not sure why I had to make this change, but otherwise compiler picks up wrong printTo definition and prints empty string for `ngraph::NodeTypeInfo` - default printTo implementation